.github/workflows/release.yml

```diff
@@ -55,6 +56,9 @@ jobs:
         run: yarn release  --ci --verbose
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Set Version Variable
+        id: set-version
+        run: echo "VERSION=$(npm pkg get version)" >> $GITHUB_ENV
```

Comment:
do we need to set this when there is already a variable VERSION_NUMBER as an output of git release action? https://github.com/NASA-IMPACT/veda-ui/blob/a100031ce6ec0df4a76c101c621b5d97d69f2648/.github/workflows/release.yml#L74